deployment/legacy-migration/maintenance-worker/migration/neo4j/users/delete.cql

```diff
@@ -1 +1,2 @@
-MATCH (n:User) DETACH DELETE n;
\ No newline at end of file
+MATCH (n:User) DETACH DELETE n;
+MATCH (e:EmailAddress) DETACH DELETE e;
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-08-16T12:35:18Z" title="Friday, August 16th 2019, 2:35:18 pm +02:00">Aug 16, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1 +1,2 @@
-MATCH (n:User) DETACH DELETE n;
\ No newline at end of file
+MATCH (n:User) DETACH DELETE n;
+MATCH (e:EmailAddress) DETACH DELETE e;
```


code creep?

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-08-16T13:02:36Z" title="Friday, August 16th 2019, 3:02:36 pm +02:00">Aug 16, 2019</time>_

---


No, you get a unique constrain violation error if you run the database import multiple times. @ulfgebhardt deletes each label separately instead of removing all at once (including in this case `EmailAddress`)